monkey/infection_monkey/credential_collectors/mimikatz_collector/mimikatz_credential_collector.py

```diff
@@ -11,12 +13,32 @@
 logger = logging.getLogger(__name__)
 
 
+MIMIKATZ_CREDENTIAL_COLLECTOR_TAG = "MimikatzCredentialsStolen"
```

Comment:
The event is already a `CredentialsStolenEvent`, we don't  need a tag that tells us credentials were stolen.

This tag should allow us to retrieve all events published by the collector. `mimikatz-credentials-collector` would be better.